vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -28,7 +28,7 @@ This tutorial covers the steps for an enterprise Entando Hub:
 
 ## Installation
 
-The Hub is installed using the Local Hub and two Entando Bundles.  The first bundle contains the micro frontends and microservices while the second sets up the initial content and pages for the Hub user experience.
+The Hub is installed using the Local Hub and two Entando Bundles. The first bundle contains the micro frontends and microservices while the second sets up the initial content and pages for the Hub user experience.
```

Comment:
experience -> interface; we can provide a consistent, tangible user interface, not user experience, i.e. focus on concrete details under our control